src/tree-internal.component.ts

```diff
@@ -98,7 +114,13 @@ export class TreeInternalComponent implements OnInit, OnChanges, OnDestroy {
       this.treeService.setController(this.tree.node.id, this.controller);
     }
 
-    this.settings = this.settings || { rootIsVisible: true };
+
+    this.settings = this.settings || { rootIsVisible: true, showCheckboxes: false, enableCheckboxes: true };
```

Comment:
we should keep default settings in https://github.com/valor-software/ng2-tree/blob/master/src/tree.types.ts#L107
This way it is consistent with single-node-level settings